content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -176,6 +176,11 @@ The following methods exist for installing kubectl on Linux:
 {{< note >}}
```

Comment:
Looks like there is already another PR #39774 that addresses issue #41442 related to improving instructions for `/etc/apt/keyrings`. PR #39774 is proposing removing this note and, instead, includes instructions to create the directory in 'Download the Google Cloud public signing key' step.

Reply:
This fix can be used until it is determined whether or not to update the main-path commands